### ISSUE TITLE
TypeError on Python3.6 runtime

### ISSUE THREAD

**Issue by pistatium** [slank/awsgi#5]
Since Lambda supported Python 3.6, I tried updating Flask's application to Python 3.6, but TypeError occurred.

```
must be str, not bytes: TypeError
Traceback (most recent call last):
File "/var/task/main.py", line 9, in lambda_handler
return awsgi.response(app, event, context)
File "/var/task/awsgi/__init__.py", line 14, in response
return sr.response(output)
File "/var/task/awsgi/__init__.py", line 35, in response
resp['body'] += chunk
TypeError: must be str, not bytes
```

resp['body'] is <class 'str'>, but output is <class 'bytes'>.
So, it has failed to combine strings.

I was not familiar with WSGI so I did not know how to concatenate it correctly, but I am glad that the error is resolved.

---
AWS Lambda Supports Python 3.6
https://aws.amazon.com/jp/about-aws/whats-new/2017/04/aws-lambda-supports-python-3-6/

**Comment by slank** [comment]
Thanks, @pistatium. To be honest, I'm not familiar enough with the string/bytes translation between Python 2 and 3 to make this work for both versions. It's something I need to learn, but I don't know when that will happen.

I'll label this as "help wanted" and see if someone else can get to sooner.

**Comment by slank** [comment]
Fixed by #6
